Sources/Gtk/Generated/AccessibleInvalidState.swift

```diff
@@ -0,0 +1,50 @@
+import CGtk
+
+/// The possible values for the %GTK_ACCESSIBLE_STATE_INVALID
+/// accessible state.
+///
+/// Note that the %GTK_ACCESSIBLE_INVALID_FALSE and
+/// %GTK_ACCESSIBLE_INVALID_TRUE have the same values
+/// as %FALSE and %TRUE.
+public enum AccessibleInvalidState: GValueRepresentableEnum {
+    public typealias GtkEnum = GtkAccessibleInvalidState
+
+    /// There are no detected errors in the value
+    case false_
```

Comment:
You could also backtick escape keywords
```swift
enum Foo {
    case `true`
}

let bar: Foo = .true
```

Reply:
I don't mind .true anymore but I can see what you mean. Just making sure you were aware of the options, it was not a change request.

Reply:
👍 Yeah thanks, I'll leave a todo in the code and decide at some point